### PR TITLE
Registering TEAM [ MangNOVA ]

### PR DESCRIPTION
**Pull request by the-cross-art**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [x] Registration Pull request.

- [] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/teams/MangNOVA/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+type: "team"                                                        
+title: DemoTeam
+name: "MagNOVA"
+github-link: "https://github.com/ "
+description: "hello I am Bot Developer"
+Team-Quote: "Transforming Technology to Transform India"
+Team_Tech_Stack: ""
```
Comment (Uyadav207) [inline content/teams/MangNOVA/index.md]: Github Link and Team tech stack not given

File: content/teams/MangNOVA/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+_**University-Name**_ : 
+
+_**country**_ :
+ 
+_**Technical-Skills**_ :
+
+_**Specialization**_ :
+
+_**GitHub-ID**_ :  
+
```
Comment (Uyadav207) [inline content/teams/MangNOVA/index.md]: Clean Left over Part

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

File: content/teams/MangNOVA/index.md (hunk 2 above)
Comment (Uyadav207) [reply content/teams/MangNOVA/index.md]: Also Check listing is to be done...
DO it and will merge your team.

**Review by Uyadav207 — APPROVED** [review]

**Comment by Uyadav207** [comment]
@the-cross-art , Change your team title to your team name ... 

title: "Your team name"

**Comment by the-cross-art** [comment]
Sir Everything good to go sir?

**Comment by the-cross-art** [comment]
Sir is everything ok now?
Are we registered successfully?

**Comment by Uyadav207** [comment]
@the-cross-art ,

Hola  👋 ! Team  you have successfully completed the `First phase of the Hackathon`. ( We have successfully merged your PR for Registration ). 
Now we Request you to join your Team to our [**CODE CAMP GROUP 1**](https://chat.whatsapp.com/EowU6RVas3QCmtEs7q7L0H).
And you can start making your project from `21th July,2020`.
We have attached important Links. 

[**TimeLine**](https://codecamphackathon.netlify.app/TimeLine/)
[**Rule Book**](https://codecamphackathon.netlify.app/RuleBook/)
[**Prizes**](https://codecamphackathon.netlify.app/Prizes/)

See you there!
@ISTESRMNCR

**Comment by the-cross-art** [comment]
Thank you Sir ,
We will give our best!

**Pull request merged**